app/models/user.rb

```diff
@@ -2,10 +2,15 @@ class User < ActiveRecord::Base
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :registerable,
-         :recoverable, :rememberable, :trackable, :validatable
+         :recoverable, :rememberable, :trackable, :validatable, :confirmable
```

Comment:
@shannonchu Shouldn't you not need the `:confirmable` module anymore since we've set it to False?